content/en/getting_started/observability_pipelines/_index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Getting Started with Observability Pipelines
+kind: Documentation
+aliases:
+    - /observability_pipelines/quickstart/
+further_reading:
+  - link: "/observability_pipelines/production_deployment_overview/"
+    tag: "Documentation"
+    text: "Production deployment overview for the Observability Pipelines Worker"
+  - link: "https://dtdg.co/d22op"
+    tag: "Learning Center"
+    text: "Safe and Secure Local Processing with Observability Pipelines"
+---
+
+## Overview
+
+The [Observability Pipelines Worker][1] can collect, process, and route logs and metrics from any source to any destination. Using Datadog, you can build and manage all of your Observability Pipelines Worker deployments at scale. 
+
+This guide walks you through deploying the Worker in your common tools cluster and configuring the Datadog Agent to send logs and metrics to the Worker.
+
+{{< img src="getting_started/observability_pipelines/dd-pipeline.png" alt="A diagram of a couple of workload clusters sending their data through the Observability Pipelines aggregator." >}}
+
+## Assumptions
+* You are already using Datadog and want to use Observability Pipelines.
+* Your services are deployed to a Kubernetes cluster in Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE).
+* You have administrative access to the cluster(s) where the Observability Pipelines Worker is going to be deployed, as well as to the workloads that are going to be aggregated.
+* You have a common tools or security cluster for your environment to which all other clusters are connected.
+
+# Prerequisites
```

Comment:
```suggestion
## Prerequisites
```

Reply:
I think we should change this to a h2. If you look at the preview site, you'll see that this heading doesn't appear in the on-page TOC.

Reply:
Oh yes, good catch!